spec/factories/edition.rb

```diff
@@ -41,6 +41,10 @@
         )
       end
 
+      if item.state == "published"
+        item.update!(published_at: item.created_at)
+      end
```

Comment:
Whitehall uses state machine code for this, so it could be rewritten as
```
item.update!(published_at: item.created_at) if item.published?
```

Reply:
Yeah, I'm not 100% happy with this being here anyway, because editions in a `superseded` or `withdrawn` state should also have a `published_at` date set. I might move it into the `live_edition` factory.